src/prefect/tiruert/submit.py

```diff
@@ -4,18 +4,225 @@
 and CARBURE API integration.
 """
 
-from datetime import date
+import os
+from datetime import date, datetime
+from string import Template
+from typing import List
 
+import pandas as pd
+import requests
+from dateutil.relativedelta import relativedelta
 from prefect import flow, task
+from prefect.states import Completed, Failed
+from sqlalchemy import text
+from sqlalchemy.orm import Session
 
+from indicators.db import get_api_db_engine, get_indicators_db_engine
+from indicators.models import IndicatorPeriod, Level
 from indicators.types import Environment
+from indicators.utils import export_indicators
+
+from .carbure import CarbureClient, CarbureUser
+
+ENERGY_BY_STATION_TEMPLATE = Template(
+    """
+    WITH
+      energy_over_period AS (
+        SELECT
+          jsonb_path_query(extras, '$$.entity') ->> 0 as entity,
+          jsonb_path_query(extras, '$$.siren') ->> 0 AS siren,
+          target as code,
+          jsonb_path_query(extras, '$$.id_station_itinerance') ->> 0 AS station,
+          jsonb_path_query(extras, '$$.energy')::NUMERIC AS energy
+        FROM
+          $environment
+        WHERE
+          code = 'tirue'
+          AND period = 'd'
+          AND (extras @> '[{"siren": "$siren"}]')
+          AND timestamp >= '$from_date'
+          AND timestamp < '$to_date'
+      )
+    SELECT
+      entity,
+      siren,
+      code,
+      station,
+      sum(energy) / 1000. AS energy_mwh
+    FROM
+      energy_over_period
+    GROUP BY
+      entity,
+      siren,
+      code,
+      station
+    ORDER BY
+      siren,
+      code,
+      station
```

Comment:
Carbure refuses total energy volumes per station lower than 1MWh, we should filter this.